share/webapp/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Cyclus WebApp Demo</title>
+  <style>
+    table, th, td {
+      border: 1px solid black;
+      border-collapse: collapse;
+      }
+    th, td {
+      padding: 5px;
```

Comment:
If someone wants to pretty up the webapp they are free to do so.  This is just a demo for testing and evaluation purposes.